functions/src/apis/bikes.ts

```diff
@@ -0,0 +1,161 @@
+import {Response as responseType} from "express";
+import {v4 as uuidv4} from "uuid";
+import {db} from "../admin";
+
+// define types
+type location = {
+  latitude: number,
+  longitude: number,
+}
+
+type entryType = {
+    bike_id: string,
+    name: string,
+    owner: string,
+    photo: string,
+    release: boolean,
+    num_ratings: number,
+    agg_rating: number,       
+    status: string,        
+    lock_combo: string,
+    location: location
+    tags: string[],
```

Comment:
Should we set the tags to enum type? ERD has 4 values: available, missing, stolen, checked out

https://www.typescriptlang.org/docs/handbook/enums.html

Reply:
That sounds like a great idea! I'd love to leverage TypeScript functionality when we can.

Reply:
I think you're thinking of the field **status** - this has the 4 values: available, missing, stolen, checked out. It's a great idea to make that an enum type. 

I think making **tags** an enum type is a great idea so that users can't put in irrelevant tags. Maybe a list of 10-20 options?